src/query/frontend/ast/pretty_print.cpp

```diff
@@ -251,6 +252,13 @@ BINARY_OPERATOR_VISIT(SubscriptOperator, "Subscript");
 
 #undef BINARY_OPERATOR_VISIT
 
+void ExpressionPrettyPrinter::Visit(RangeOperator &op) {
```

Comment:
How do we reach this code/see its effect?

Reply:
This is used when outputting the plan as a JSON.
The JSON is sent as part of the summary when running an EXPLAIN query.

Query (with index):
```
EXPLAIN MATCH (n:l) WHERE 1 < n.p < 10 RETURN n;
```
Explain query result:
```
+---------------------------------------------+
| QUERY PLAN                                  |
+---------------------------------------------+
| " * Produce {n}"                            |
| " * ScanAllByLabelPropertyRange (n :l {p})" |
| " * Once"                                   |
+---------------------------------------------+
```
JSON:
```
"explain": {
  "input":{
    "input":{
      "name":"Once"
    },
    "label":"l",
    "lower_bound":{
      "type":"exclusive",
      "value":"(ParameterLookup 7)"
    },
    "name":"ScanAllByLabelPropertyRange",
    "output_symbol":"n",
    "property":"p",
    "upper_bound":{
      "type":"exclusive",
      "value":"(ParameterLookup 13)"
    }
  }
}
```